components/learningUnits/LearningUnitInfo.js

```diff
@@ -16,15 +17,15 @@ export default function LearningUnitInformation({ learningUnitId }) {
     return <Alert severity="error">Error</Alert>;
   }
 
-  const goBackPageText = "Back to curriculum";
-
   return (
     <Card>
       <CardContent>
-        <PreviousPage goBackText={goBackPageText} />
-        <Typography variant="h2" component="div">
-          {learningUnit.name}
-        </Typography>
+        <div className={styles.containerTitulo}>
```

Comment:
Check if this can be done with the props that the material components receive.

Reply:
I searched and found that style is needed to arrange the position of two or more elements. What can be done different is that instead of changing styles in the css files, the style can be passed through a `Container` component and the `sx` prop, for example, `<Container sx={ {display: "flex", justifyContent: "space-between"} }>`

Reply:
I think that's what we've been doing, I think it suits better the way we have been coding

Reply:
Done!